pkg/cmd/connector/use/use.go

```diff
@@ -101,6 +89,18 @@ func runUse(opts *options) error {
 		return err
 	}
 
+	if opts.interactive {
+		// run the use command interactively
+		err = runInteractivePrompt(opts, &conn)
+		if err != nil {
+			return err
+		}
+		// no Kafka was connector, exit program
```

Comment:
> "no Kafka was connector" 

but Connector, no Kafka can be, Yes..

![image](https://user-images.githubusercontent.com/981838/177986472-bcd4a0fd-2dc7-43f6-959f-5ed89dc9c386.png)